### DISCUSSION TITLE
Can't find Scouts on the App Store

### DISCUSSION THREAD

**Discussion by Tpman** [airsdk/Adobe-Runtime-Support#771] [General]
I can't find the adobe scout app on the app store.
It's still on the Android Play Store.
How do I use adobe scout in ios?

**Comment by htmiel** [comment]
Does this work for you? https://apps.apple.com/ro/app/adobe-scout/id562450748

**Reply by Tpman** [reply]
The link works fine. Thank.
This is a fact I learned through link execution, but it was not that there was no app when searching in the App Store.
I don't know why, but I thought there was no app because it was pushed back a lot from the searched list.

**Reply by htmiel** [reply]
I never find it either with the search, but I look for another app by Adobe, then go to all apps by that developer, and there I can find it.
